sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1626,6 +1663,13 @@ class DataSourceV2SQLSuite
     assert(e.message.contains(s"$sqlCommand is only supported with v1 tables"))
   }
 
+  private def testV1CommandNamespace(sqlCommand: String, namespace: String): Unit = {
```

Comment:
Since this is very specific to your test, can you define it inside your test function? You can look at `checkFailure`, etc. in this file for an example.

Reply:
Ok, good idea